quipucords/tests/scanner/openshift/test_api.py

```diff
@@ -178,6 +186,18 @@ def test_retrieve_operators(ocp_client: OpenShiftApi):
     assert isinstance(operators[-1], LifecycleOperator)
 
 
+@pytest.mark.vcr(
+    VCRCassettes.OCP_ACM,
+    VCRCassettes.OCP_DISCOVERER_CACHE,
+)
+def test_retrieve_acm_metrics(ocp_client: OpenShiftApi):
+    """Test retrieve acm metrics method."""
+    # OCP instance used on VCR preparation should have ACM operator installed,
+    # with active multiClusterHub instance
+    acm_metrics = ocp_client.retrieve_acm_metrics()
+    assert isinstance(acm_metrics, list)
```

Comment:
I just realized now this function will need more proper testing since we are not using a dedicated entity for it.